embrace-android-sdk/src/test/java/io/embrace/android/embracesdk/arch/EmbraceAttributeExtensions.kt

```diff
@@ -60,6 +62,36 @@ internal fun EmbraceSpanData.assertSuccessful() {
     assertNull(attributes[ErrorCodeAttribute.Failure.otelAttributeName()])
 }
 
+internal fun Span.assertIsTypePerformance() = assertIsType(EmbType.Performance.Default)
```

Comment:
Duplicate the extension functions for testing purposes that exist for `EmbraceSpanData`. Hopefully we can delete that soon once we delete v1